index.js

```diff
@@ -82,17 +84,19 @@ const effects = {
 		}
 
 		let result = chunks.join('');
-		if (Math.random() > 0.99) {
+		const mr = Math.random();
```

Comment:
I don't see how adding a new line and a new variable is useful, as the lines are already short and readable.